tests/preprocessing/test_generator.py

```diff
@@ -262,7 +262,7 @@ def test_complete(self):
         simple_generator = SimpleGenerator(input_bboxes_group, input_labels_group, image=input_image, num_classes=6)
         # expect a UserWarning
         with pytest.warns(UserWarning):
-            _, [_, labels_batch] = simple_generator.next()
+            _, [_, labels_batch] = simple_generator.__getitem__(0)
```

Comment:
Shouldn't this just be `simple_generator[0]`?

Reply:
The tests don't get that far yet because of the style checks. Might make sense for us to split that into two separate tests so the functional bits can be checked even if the style bit gives errors.